### PR TITLE
Add justification document for tf_42045

### PR DESCRIPTION
**Pull request by fridex**

### CODE REVIEW

File: _j/tf_42045.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+---
+title: TensorFlow in version 2.3 produces wrong model accuracy when the model is accuracy
```
Comment (pacospace) [inline _j/tf_42045.md:2]: when the model `is serialized`?

File: _j/tf_42045.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+TensorFlow in version 2.3 produces wrong model accuracy when the model is
+serialized using `accuracy`, use `sparce_categorical_accuracy` instead
+
+## Issue description
+
+TensorFlow in version 2.3 fails to serialize the model properly. This resolves
+in bad model accuracy when the model is loaded back. The issue is caused by a
+bug in `accuracy` which has been already [resolved in more recent versions][1].
+
+To avoid this issue, make sure you use `sparce_categorical_accuracy` instead if
```
Comment (pacospace) [inline _j/tf_42045.md:14]: instead `of`

Comment (fridex) [reply _j/tf_42045.md]: Fixed in 38bfa29d

File: _j/tf_42045.md (hunk 1 above)
Comment (fridex) [reply _j/tf_42045.md]: Fixed in 38bfa29d

**Review by pacospace — APPROVED** [review]
Thanks :100:

**Pull request merged**